horovod/run/common/util/env.py

```diff
@@ -23,3 +24,18 @@
 
 def is_exportable(v):
     return not any(re.match(r, v) for r in IGNORE_REGEXES)
+
+
+def get_env_rank_and_size():
+    rank_env = ['HOROVOD_RANK', 'OMPI_COMM_WORLD_RANK', 'PMI_RANK']
+    size_env = ['HOROVOD_SIZE', 'OMPI_COMM_WORLD_SIZE', 'PMI_SIZE']
+
+    for rank_var, size_var in zip(rank_env, size_env):
+        rank = os.environ.get(rank_var)
+        size = os.environ.get(size_var)
+        if rank is not None and size is not None:
```

Comment:
if only one is defined, shall we throw an error?

Reply:
Not sure. CC @tgaddair